awx/ui_next/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.jsx

```diff
@@ -55,7 +58,7 @@ function WorkflowJobTemplateDetail({ template, i18n, webHookKey }) {
       )}
       {template.webhook_service && (
         <TextListItem component={TextListItemVariants.li}>
-          {i18n._(t`- Webhooks`)}
+          {i18n._(t`- Enable Webhook`)}
```

Comment:
On the add WF template form, when you are creating a Webhook configuration, the Webhook URL and Webhook Key fields are missing. These should be visible, but have the pending status, like so:

![Screenshot from 2020-03-04 09-40-31](https://user-images.githubusercontent.com/19942822/75890324-345eef00-5dfc-11ea-9a97-4a8b8d4a48f7.png)

Reply:
Yea in talking with @AlexSCorey I think this was done intentionally on the add form since those fields are generated by the backend.  In limited testing of the old UI I think it behaves the same way(?).  If so, lets file a follow-up enhancement for the UX pattern here.

Reply:
@dsesami one of the problems with have the url field visible in this state is that the id of the template is required to generate an accurate url.  Without the id the url will be something like `/api/v2/workflow_job_template/undefined/github` and I was of the opinion that was not good form, and it doesn't follow the pattern in the old ui.  In the old ui both of those fields are not displayed on the add form.  

In either case, if this is necessary I agree with Mike, let's make a follow up issue.

Reply:
Yeah, I made a similar suggestion with the old UI. I'll create a follow-up ticket.

Reply:
ticket here: https://github.com/ansible/awx/issues/6162